unittests.py

```diff
@@ -42,6 +42,12 @@ def test_alive_greater_than_150(self):
     def test_dead_greater_than_150(self):
         self.assertFalse(self.verifier(examples.examplePersonDeadOver150))
 
+class TestForBigamy(unittest.TestCase):
+    def testMarriedPerson(self):

```

Comment:
You have test cases but `main` in project03 should be verifying the input gedcom file with these too